docs/book/src/topics/consuming-existing-azure-infrastructure.md

```diff
@@ -0,0 +1,6 @@
+# Consuming Existing Azure Infrastructure
+
+Normally, Cluster API will create infrastructure on Azure when standing up a new workload cluster. However, it is possible to have Cluster API re-use existing Azure infrastructure instead of creating its own infrastructure.
+
+CAPZ supports [externally managed cluster infrastructure](https://github.com/kubernetes-sigs/cluster-api/blob/10d89ceca938e4d3d94a1d1c2b60515bcdf39829/docs/proposals/20210203-externally-managed-cluster-infrastructure.md).
+If the `AzureCluster` resource includes a "cluster.x-k8s.io/managed-by" annotation then the [controller will skip any reconciliation](https://cluster-api.sigs.k8s.io/developer/providers/cluster-infrastructure.html#normal-resource).
```

Comment:
Are there any recommendations we need to give to folks for using this? something like "you should only use this feature if ..." ? Don't they need their own controllers?